src/Authentication/Authentication/Common/GraphSessionInitializer.cs

```diff
@@ -27,5 +31,34 @@ internal static GraphSession CreateInstance(IDataStore dataStore = null)
                 DataStore = dataStore ?? new DiskDataStore()
             };
         }
+        /// <summary>
+        /// Initializes <see cref="GraphSession"/>. with Output via Cmdlet methods
+        /// </summary>
+        /// <param name="cmdLet"></param>
+        public static void InitializeSession(PSCmdlet cmdLet)
```

Comment:
We should also remove `public static void InitializeSession()` method in [line 18](https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/668/files#diff-b8fd14ba87f272ebb3b25a6c432fddc2666b0f20e8880c2558cd97e44feff1baR18-R21) since it has been replaced this method and is no longer being used.

Reply:
Changed it slightly from your recommendation to enable passing a custom writer.